cyclonedx/model/service.py

```diff
@@ -259,6 +261,10 @@ def data(self) -> 'SortedSet[DataClassification]':
 
     @data.setter
     def data(self, data: Iterable[DataClassification]) -> None:
+        if data:
+            warn('`@.data` is deprecated from CycloneDX v1.5 onwards. '
```

Comment:
revertred. adding new warnings is not in the scope of the ticket, nor shall it be part of this very PR.

please create a new ticket/PR for this topic.